aspnetcore/security/docker-https.md

```diff
@@ -52,23 +52,22 @@ Use the following instructions for your operating system configuration.
 Generate certificate and configure local machine:
 
 ```dotnetcli
-dotnet dev-certs https -ep %USERPROFILE%\.aspnet\https\aspnetapp.pfx -p <password>
+dotnet dev-certs https -ep %USERPROFILE%\.aspnet\https\aspnetapp.pfx -p <$CREDENTIAL_PLACEHOLDER$>
```

Comment:
Can we get rid of the dollar signs? They don't add any value and could be confusing to readers. `<CREDENTIAL_PLACEHOLDER>`

Reply:
> Can we get rid of the dollar signs? They don't add any value and could be confusing to readers. `<CREDENTIAL_PLACEHOLDER>`

Done, See [this commit](https://github.com/dotnet/AspNetCore.Docs/pull/27186/commits/2021e914e151fc1211305fb5b2cef5277e4edd6a)